### PR TITLE
Tests based on NumPy version

### PR DESCRIPTION
**Pull request by newlawrence**
It is now possible to specify the **NumPy** version in the continuous integration build scripts.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -28,18 +31,18 @@ install:
...
-  - "conda install -q numpy numba>=0.18 astropy>=1.0 matplotlib pytest scipy"
+  - "conda install -q numpy=%NUMPY% numba>=0.18 astropy>=1.0 matplotlib pytest scipy"
   - "conda install jplephem -c poliastro"
   - "pip install -e ."    # Needed to use py.test properly
   
 build: off
 
 test_script:
-  - "py.test poliastro"
+  - "py.test poliastro"
```
Comment (Juanlu001) [inline appveyor.yml]: Nitpicking: you forgot a newline character here! :sweat_smile:

**Comment by Juanlu001** [comment]
Thanks again for taking care of the infrastructure issues, I definitely owe you a beer or a whole dinner :)

Do you know if there's a way to trigger the Appveyor and Anaconda builds for this branch? Just to make sure everything goes well before we merge.

**Comment by newlawrence** [comment]
You're welcome 😊. But your project is awesome and deserves the best infrastructure to reach everyone!

It's strange that pull requests aren't triggered automatically, I remembered from our last experience that it should work. I imagine you've not disabled pull request submitting from each site configuration, so maybe if you try [this](http://stackoverflow.com/questions/32607885/appveyor-doesnt-update-github-pull-request-with-the-build-status)... revoking authorization and granting it again may resolve the problem.

**Comment by Juanlu001** [comment]
Fine, apparently there's a bit of a mess because Appveyor does not support organizations and I used a separate email account to setup the integration. I'm working on it.

In the meanwhile, Anaconda hooks are failing too, so I still have a bit of work to do.

**Comment by Juanlu001** [comment]
In principle we should be set now @newlawrence, please push another commit (use it to fix that tiny whitespace issue) to see if everything works.

**Comment by Juanlu001** [comment]
Or better: I'll try to close and reopen, in case it works.

**Comment by newlawrence** [comment]
Missing **Anaconda Cloud** yet.

**Comment by Juanlu001** [comment]
Not sure if building on pull requests is possible on Anaconda Cloud, plus it's Friday evening so... :shipit:

**Comment by Juanlu001** [comment]
Thanks again @newlawrence!

**Comment by newlawrence** [comment]
One more thing, remember to change the [following lines](https://github.com/newlawrence/poliastro/commit/8654dbc26c27858823da008486956384c51c939f) in the next commit, or **Travis** and **Appveyor** will fail.

As I uploaded more **jplephem** versions to your [repo](https://anaconda.org/poliastro/jplephem/files), it will always try to use the one that relies on the newest version of **NumPy**, and since that version (1.10) isn't supported yet, all the combinations will fail (except for **Python 3.3** that has no **NumPy 1.10** available).

This lines forces **conda** not to upgrade **NumPy** and use the version specified in the script.

**Pull request merged**